qt/aqt/addons.py

```diff
@@ -46,6 +46,10 @@
 )
 
 
+class AbortAddonImport(Exception):
```

Comment:
Wasn't sure about the naming and whether or not to put `Excepetion` somewhere in there. This felt the most semantical